format_implementing.md

```diff
@@ -0,0 +1,92 @@
+# Format Implementation Guide
+
+If you're a launcher developer, this document can help you by providing useful tips and recommended practices for implementing the MODIP Format.
+
+---
+
+## Rendering HTML
+
+The MODIP Format Specification allows the usage of HTML in certain fields, such as `description` and `changelog`. HTML is widely used and meets many of the requirements needed for a field like `description`. However, HTML can be **very dangerous**. Take caution when displaying content using HTML. It's recommended to sanitize your HTML according to the rules listed below.
+
+### Disallowed Tags
+
+You should never attempt to render any of the following tags: `script`, `style`, `link`, `title`, `button`, `form`, `input`, `meta`, `option`, `textarea`, and `select`.
```

Comment:
Blacklists don't work. Ever.

Reply:
&lt;img src="blah://foo" onerror="alert('pwned');">

Reply:
&lt;scr&lt;script>ipt> is also a common bug. When you remove the &lt;script> there's still a &lt;script>

Reply:
Yeah you’re right. If HTML is going to be sanitized, it should be a whitelist. This does run into the whole issue of whether HTML should be used at all.